core/src/main/java/fr/sncf/osrd/api/PathfindingRoutesEndpoint.java

```diff
@@ -221,8 +215,7 @@ public Response act(Request req) throws IOException {
             }
             return new RsJson(new RsWithBody(adapterResult.toJson(res)));
         } catch (Throwable ex) {
```

Comment:
you can add multiple catch clauses, one for OSRDException, and one catch-all for RuntimeException. this alleviates the need for an auxiliary function